### PR TITLE
added toast for adding receipts to expenses

### PR DESCRIPTION
**Pull request by sandy-fyle**
added toast for adding receipts to expenses
before
after
image

### CODE REVIEW

**Review by Dimple16 — CHANGES_REQUESTED** [review]
you've made changes to a common component, can you add screenshots of how the change looks on other pages too? are those tested?

File: src/app/shared/components/receipt-preview-thumbnail/receipt-preview-thumbnail.component.ts [hunk 1]
```diff
@@ -32,14 +34,20 @@ export class ReceiptPreviewThumbnailComponent implements OnInit, DoCheck {
...
+    private trackingService: TrackingService
+  ) {}
 
   ngOnInit() {
     this.sliderOptions = {
       slidesPerView: 1,
       spaceBetween: 80,
     };
     this.previousCount = this.attachments.length;
+    console.log(this.attachments.length);
```
Comment (Dimple16) [inline src/app/shared/components/receipt-preview-thumbnail/receipt-preview-thumbnail.component.ts:49]: remove logs

**Comment by Dimple16** [comment]
can you check if we have a toast on adding receipts from my expenses page? Ideally, we should have a toast there too

**Review by Dimple16 — CHANGES_REQUESTED** [review]

**Comment by sandy-fyle** [comment]
> can you check if we have a toast on adding receipts from my expenses page? Ideally, we should have a toast there too

its not there, looking into it.

**Comment by sandy-fyle** [comment]
> can you check if we have a toast on adding receipts from my expenses page? Ideally, we should have a toast there too

I have made the changes now its showing on my expenses page too
image

File: src/app/shared/components/receipt-preview-thumbnail/receipt-preview-thumbnail.component.ts [hunk 2]
```diff
@@ -3,6 +3,10 @@ import { timer } from 'rxjs';
 import { FileObject } from 'src/app/core/models/file_obj.model';
 import { Swiper } from 'swiper';
 import { SwiperComponent } from 'swiper/angular';
+import { MatSnackBar } from '@angular/material/snack-bar';
```
Comment (Dimple16) [inline src/app/shared/components/receipt-preview-thumbnail/receipt-preview-thumbnail.component.ts:6]: remove imports if not used

File: src/app/shared/components/receipt-preview-thumbnail/receipt-preview-thumbnail.component.ts [hunk 3]
```diff
@@ -32,7 +34,11 @@ export class ReceiptPreviewThumbnailComponent implements OnInit, DoCheck {
 
   numLoadedImage = 0;
 
-  constructor() {}
+  constructor(
+    private matSnackBar: MatSnackBar,
```
Comment (Dimple16) [inline src/app/shared/components/receipt-preview-thumbnail/receipt-preview-thumbnail.component.ts:38]: remove if unused

**Review by Dimple16 — CHANGES_REQUESTED** [review]

File: src/app/shared/components/receipt-preview-thumbnail/receipt-preview-thumbnail.component.ts [hunk 4]
```diff
@@ -18,8 +17,6 @@ export class ReceiptPreviewThumbnailComponent implements OnInit, DoCheck {
 
   @Input() canEdit: boolean;
 
-  @Input() hideLabel: boolean;
```
Comment (Dimple16) [inline src/app/shared/components/receipt-preview-thumbnail/receipt-preview-thumbnail.component.ts:21]: why are we removing this?

**Review by Dimple16 — CHANGES_REQUESTED** [review]

File: src/app/shared/components/receipt-preview-thumbnail/receipt-preview-thumbnail.component.ts (hunk 4 above)
Comment (sandy-fyle) [reply src/app/shared/components/receipt-preview-thumbnail/receipt-preview-thumbnail.component.ts:21]: it wasn't getting used anywhere

Comment (sandy-fyle) [reply src/app/shared/components/receipt-preview-thumbnail/receipt-preview-thumbnail.component.ts:21]: Should I add it back?

Comment (sandy-fyle) [reply src/app/shared/components/receipt-preview-thumbnail/receipt-preview-thumbnail.component.ts]: oh sorry! it was a misunderstanding, added it back

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**